source/_patterns/00-atoms/components/info-bar.yaml

```diff
@@ -8,11 +8,28 @@ properties:
     type: string
     enum:
       - attention
+      - dismissible
```

Comment:
this is not necessary anymore with the separate `dismiss` property?

Reply:
Assuming absence of the property means not dismissible, and presence of the property means dismissible; this makes one of the `cookie*` properties mandatory, which is in fact captured in this schema

Reply:
This value is [used in the css selectors](https://github.com/elifesciences/pattern-library/pull/958/files#diff-007b18257254acbb20bed8144ad26566R78).